pkg/plugins/discovery_test.go

```diff
@@ -79,9 +80,6 @@ func TestFileSpecPlugin(t *testing.T) {
 	}
 
 	for _, c := range cases {
-		if err = os.MkdirAll(path.Dir(c.path), 0755); err != nil {
```

Comment:
yeah, the first implementation allowed to have specs inside directories. But it's fine to remove it.